docs/services/flows.rst

```diff
@@ -3,6 +3,26 @@ Globus Flows
 
 .. currentmodule:: globus_sdk
 
+The Flows service allows users to create automation workflows or, simply, "flows".
+When a flow is started, it must be authorized to perform actions on the user's behalf.
+
+Because a running flow can perform actions on the user's behalf,
+the Globus SDK has two client classes that can interact with the Flows service:
+a :class:`FlowsClient` and a :class:`SpecificFlowClient`.
+They differ in what scopes and user consents they require:
+
+*   The :class:`FlowsClient` is able to create, update, and delete flows.
+    It is also able to gather information about flows and runs.
+
+    Users must consent to allow the client application to administer flows and runs.
+
+*   The :class:`SpecificFlowClient` is able to start a specific flow.
```

Comment:
Might be worth adding a bit of additional clarifying copy here, e.g., "Some run-related operations require a special client that is associated with the flow used to start that run. Instantiating a `SpecificFlowClient` using the ID of that flow enables it to perform operations that authorize a flow to invoke actions on behalf of the user in the course of a run." (This could maybe be refined further, but hopefully is in the ballpark?)

Might also want to note that in some cases, a user will need to use both clients in the course of their application.

Reply:
I think I've addressed this.